desktop/ponyprog.desktop

```diff
@@ -1,12 +1,12 @@
 [Desktop Entry]
-Version=3.0.0
+Version=3.0
 Name=Ponyprog
 Comment=EEPROM and microcontroller flasher
 GenericName=Ponyprog
```

Comment:
This 'generic name' doesn't make much sense as it's not generic. I think it'd make sense to drop it.